src/tests/intellij/test_analysis.py

```diff
@@ -27,4 +27,5 @@ def test_analysis_intellij(configurations, setup_intellij, app_name, analysis_da
         intellij.verify_story_points(
             html_file_location=html_file_location,
             expected_story_points=expected_story_points,
+            legacy="legacyReports" in application_data["options"],
```

Comment:
@midays , thanks for the PR .

For VSCode, 
1) Now that legacy reports are not default anymore, 'legacyReports' would be an advanced option .The test for legacy reports should go into src/tests/<IDE>/test_advanced_options.py . 
2) Additionally, the item that will be used for the legacy reports tests in src/data/analysis.json needs to be updated with "legacyReports": "true".

Could you clarify if the behavior is the same for IntelliJ ?
This behavior is for VSCode. Do let me kno

Reply:
yes it's the same as in vscode